docs/pages/docs/api-reference/generative-ui/use-actions.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: useActions
+layout:
+  toc: false
+---
+
+import { Tabs, Tab } from 'nextra-theme-docs';
+
+# useActions
+
+## `useActions`
+
+`useActions` is a hook to help you access your Server Actions from the client.
```

Comment:
```suggestion
`useActions` is a hook to help you access your Server Actions from the client. It’s required to access these server actions via this hook (because we patched these actions when they’re passing through this context), and if you directly import the actions instead of using useActions, you might run into the current issue of “cannot find client component.”
```